cerebrium/development/serve.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Served sandbox with hot-reload
+description: Use the `cerebrium serve` command to rapidly iterate on your code
+---
+
+When you are developing a `cortex` deployment on **Cerebrium**, waiting for a build to complete can be time-consuming. To speed up your development process, you can use the `cerebrium serve` command to rapidly iterate on your deployment.
+
+This allows you to run your deployment on a dedicated server, and see the results of your changes in a few seconds.
+
+<Note>
+  This feature is currently in beta and is available to all users. As such, you
+  may encounter bugs or limitations. We are actively working on improving the
+  experience and adding more features. If you have any feedback or suggestions,
+  we'd love to hear from you on discord or slack!
+</Note>
+
+## Usage
+
+To start a served instance, first navigate to the root folder of your **cortex deployment**.
+
+Then, simply run the following command in your terminal:
+
+```bash
+cerebrium serve start
+```
+
+After running the `cerebrium serve start` command, you will see output in your terminal indicating that the server is starting up. This output will include a URL that you can use to access your deployment locally.
+
+Here are some additional commands that can be useful when working with `cerebrium serve`:
```

Comment:
I would make this sequential - ie: after talking about start, mention the local API server that gets created. Don't mention all the details below but rather that a local endpoint is created that they can make a request to and it mimics a endpoint once deployed.

Then mention as they save files, it syncs changes and lastly stop.